src/test/rpc/TransactionEntry_test.cpp

```diff
@@ -66,7 +66,7 @@ class TransactionEntry_test : public beast::unit_test::suite
             auto const result =
                 env.client().invoke("transaction_entry", params)[jss::result];
             BEAST_EXPECT(!result[jss::ledger_hash].asString().empty());
-            BEAST_EXPECT(result[jss::error] == "transactionNotFound");
+            BEAST_EXPECT(result[jss::error] == "malformedRequest");
```

Comment:
I think we want to give a valid (but not present) value for the hash, rather than `DEADBEEF` or we're changing what this test is testing.